content/design-systems-for-developers/react/en/test.md

```diff
@@ -37,9 +37,11 @@ Visual tests capture an image of every UI component in a consistent browser envi
 
 ![Visual test components](/design-systems-for-developers/component-visual-testing.gif)
 
-If you’re building a modern UI, visual testing saves your frontend team from time-consuming manual review and prevents expensive UI regressions. We’ll demo visual testing using Chromatic, an industrial-grade service by the Storybook maintainers.
+If you’re building a modern UI, visual testing saves your frontend team from time-consuming manual review and prevents expensive UI regressions.
 
-First, go to [chromatic.com](https://chromatic.com) and sign up with your GitHub account.
+In the <a href="/design-systems-for-developers/react/en/review/#publish-storybook"> previous chapter</a> we've learned how to publish our Storybook and get feedback. We'll expand on what we've learned and demo visual testing using Chromatic, an industrial-grade service by the Storybook maintainers.
```

Comment:
Can we make this URL relative so that we don't have to update it for every framework/translation?

Reply:
To the best of my knowledge with markdown you cannot do it. With MDX we might achieve this. That's something that would need a bit more of testing.